config/config.go

```diff
@@ -388,6 +388,17 @@ type Local struct {
 	// BlockServiceCustomFallbackEndpoints is empty.
 	// The archiver is randomly selected, if none is available, will return StatusNotFound (404).
 	EnableBlockServiceFallbackToArchiver bool `version[16]:"true"`
+
+	// CatchupBlockValidateMode defines the degree of validation used by the catchup service. This field is a bit-field, where the
```

Comment:
Suggested rewording of the comments:

CatchupBlockValidateMode is a development and testing configuration used by the catchup service. 
It can be used to omit certain validations to speed up the catchup process, or to apply extra validations which are redundant in normal operation.
This field is a bit-field with:
bit 0: (default 0) 0: verify the block certificate; 1: skip this validation
bit 1: (default 0) 0: verify payset committed hash in block header matches payset hash; 1: skip this validation
bit 2: (default 0) 0: don't verify the transaction signatures on the block are valid; 1: skip this redundant validation
bit 3: (default 0) 0: don't verify that the hash of the recomputed payset matches the hash of the payset committed in the block header; 1: skip this redundant validation
Note: not all permutations of the above bitset are currently functional. In particular, the ones that are functional are:
0 : default behavior.
3: speed up catchup by skipping necessary validations
12 : perform all validation methods (normal and additional)